elephant/spectral.py

```diff
@@ -523,6 +529,543 @@ def multitaper_psd(signal, n_segments=1, len_segment=None,
     return freqs, psd
 
 
+def multitaper_cross_spectrum(signals, fs=1., nw=4, num_tapers=None,
+                              peak_resolution=None, return_onesided=True,
+                              attach_units=True):
+    """
+    Estimates the cross spectrum of a given 'neo.AnalogSignal' using the
+    Multitaper method.
+
+    The cross spectrum is obtained through the following steps
+
+    1. obtain approximately independent estimates of the spectrum for each
+       signal by multiplying the segment with tapering functions (obtained as
+       the discrete prolate spheroidal functions, also known as slepian
+       functions) and calculating the cross spectrum for the tapered signals.
+       The number of tapering functions (and hence the number of the estimates)
+       is specified by the parameter `num_tapers`.
+
+    2. average the approximately independent estimates of each signal to
+       decrease overall variance of the estimates
+
+    Parameters
+    ----------
+    signal : neo.AnalogSignal or pq.Quantity or np.ndarray
+        Time series data of which PSD is estimated. When `signal` is
+        np.ndarray, it needs to be passed as a 2-dimensional array of shape
+        (n_channels, n_samples), and the data sampling frequency should be
+        given through the keyword argument `fs`.
+    fs : float, optional
+        Specifies the sampling frequency of the input time series
+        Default: 1.0.
```

Comment:
Default lines are not ended by period, to avoid confusion with floats

Reply:
Addressed in https://github.com/NeuralEnsemble/elephant/pull/525/commits/98baf49dea1b9406797487bdd3e93e46aa2e0d4b